src/StringToVectorConverter.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef STRINGTOVECTORCONVERTER_H
+#define STRINGTOVECTORCONVERTER_H
+
+#include <string>
+#include <vector>
+#include <algorithm>
+
+std::vector<int> ConvertStringToVector(std::string str){
+
+    str.erase(std::remove(str.begin(), str.end(), '|'), str.end());
+
+    for(int i = 0; i < str.length(); i ++)
+        if(str[i] == '/')
+            str[i] = (10-(str[i-1]-48))+48;
```

Comment:
-48 and +48 may be misleading. std::stoi and std::to_string are more descriptive.

Reply:
I have a bit of a problem here. std::stoi is returning int from passed string. But taking one character from the string is type of char so compiler during build returns: invalid conversion error. I don't know how to resolve it. Solution for now is working but still calculate with ASCII code. Could you advise something?

Reply:
Ah, ok. So try std::atoi :)